extension/src/requestHandlers/createOrder.ts

```diff
@@ -142,12 +142,13 @@ export function getCreateOrderParams(ctPayment: CTPayment, cart: CTCart): Promis
     const [paymentMethod, paymentIssuer] = ctPayment.paymentMethodInfo.method.split(',');
     const parsedCtPayment = JSON.parse(ctPayment.custom?.fields?.createPayment || '{}');
     const locale = parsedCtPayment.locale || configLocale;
+    const fallbackEmail = cart.customerEmail || cart.customer?.email;
```

Comment:
Look below for a comment on types, first need to update the query to get the customer populated on cart (if possible on REST API)..